package.json

```diff
@@ -1,20 +1,32 @@
 {
-  "name": "vui-table",
+  "name": "d2l-table",
   "version": "0.1.0",
   "description": "Mixins and CSS for applying table styles",
   "scripts": {
     "autoprefix": "postcss -c postcss.config.json && postcss -c ./test/postcss.config.json",
-    "clean": "rimraf *.css && rimraf ./test/*.css",
+    "clean": "rimraf *.css ./test/*.css dumps/ reports/",
     "prebuild": "npm run clean && imgtoscss -p vui- -o table-icons.scss *.svg",
```

Comment:
Hopefully this build step to generate the icons SCSS can go away? (and the dependencies it needed)